services/housetables/src/main/java/com/linkedin/openhouse/housetables/api/validator/impl/OpenHouseUserTableHtsApiValidator.java

```diff
@@ -51,6 +51,14 @@ public void validateDeleteEntity(UserTableKey userTableKey) {
   public void validateGetEntities(UserTable userTable) {
     List<String> validationFailures = new ArrayList<>();
 
+    // This will be removed when we start to support general filters
+    if (!(userTable.getTableVersion() == null
```

Comment:
What this filter condition is doing?

Reply:
+1. The condition does not seem intuitive

Reply:
It means all other fields are null. We don't have any use case to search by other fields so I want to restrict the capability of this API for now.